vcd/provider.go

```diff
@@ -246,6 +246,13 @@ func Provider() *schema.Provider {
 				Description: "The API token used instead of username/password for VCD API operations. (Requires VCD 10.3.1+)",
 			},
 
+			"service_account_token_file": {
```

Comment:
I think we can leave it in case there are users that use multiple fields and only change the `auth_type` as ideally we should have that for all fields but that could possibly be breaking.

Reply:
ok, let it be